src/styles/_styles.scss

```diff
@@ -25,3 +25,12 @@
     }
   }
 }
+
+:global(.playkit-player) {
+  :global(.playkit-player .playkit-top-bar .playkit-right-controls) {
+    display: flex;
+  }
```

Comment:
dual-screen plugin doesn't have icon, so styles for `.playkit-right-controls` must be removed from PR